.github/workflows/test.yml

```diff
@@ -198,14 +198,12 @@ jobs:
           nix build .#hs-backend-booster:exe:rpc-client --extra-experimental-features 'nix-command flakes' --print-build-logs
 
       - name: Run unit tests
-        if: ${{ matrix.os != 'macos-12' }}
```

Comment:
not sure its worth enabling these on the public runners? we already test on the M1 which is a lot faster

Reply:
I was just wondering why we aren't running the tests. If there is a good reason not to we can revert that part.

Reply:
As discussed, integration tests take very long on the public runner - those are left disabled now.